core/domain/skill_services.py

```diff
@@ -409,19 +407,21 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
     - committer_id: str. The id of the user who is performing the update
         action.
     - skill_id: str. The skill id.
-    - change_list: list of dicts, each representing a SkillChange object.
+    - change_list: list(SkillChange). Each element is a SkillChange object.
         These changes are applied in sequence to produce the resulting
         skill.
     - commit_message: str or None. A description of changes made to the
         skill. For published skills, this must be present; for
         unpublished skills, it may be equal to None.
+
+    Raises:
+        ValueError: Expected commit message.
```

Comment:
ValueError: No commit message was provided.

(The description of the error should explain what circumstances cause the error to occur.)

Reply:
Done.